src/dense/interpolants.jl

```diff
@@ -14,10 +14,11 @@ end
 Hairer Norsett Wanner Solving Ordinary Differential Euations I - Nonstiff Problems Page 192
 """
 @def dp5pre0 begin
-    b10 = @evalpoly(Θ, 0, 1)
-    b20 = @evalpoly(Θ, 0, 1, -1)
-    b30 = @evalpoly(Θ, 0, 0, 1, -1)
-    b40 = @evalpoly(Θ, 0, 0, 1, -2, 1)
+    Θ² = Θ * Θ
+    b10 = Θ * @evalpoly(Θ, 1)
```

Comment:
That's just
```suggestion
    b10 = Θ
```
isn't it?